packages/css-engine/src/core/style-sheet.ts

```diff
@@ -88,7 +88,13 @@ export class StyleSheet {
   markAsDirty() {
     this.#isDirty = true;
   }
-  get cssText() {
+  generateWith({
```

Comment:
regarding naming, I used toString in all rules to get a css string from a model with a .cssText shorthand when you don't need options

Would be great to keep it consistent here and in the nesting rule the same

Reply:
toString in js is not suppose to accept any input data. So it may be confusing. It is confusing for me at some point. It's like `''.toString({ content: 'content' })`

Reply:
then lets rename all of those toString in all rules to toCss or something

Reply:
in other PR